### PR TITLE
Added Prisma model for Traces

### PR DESCRIPTION
**Pull request by justinliang1020**
Promptable UI now fetches from database for traces.

Seed the example traces using "npx prisma db seed".

The endpoint for receiving traces is localhost:3000/api/traces, if you pass the trace json as the body of a PUT response it should update the database.

TODO: i tried running the tracing-web example but don't think it's currently sending anything to the endpoint since the "sendTraceToServer" function in promptable/src/tracing.ts isn't called.

### CODE REVIEW

File: packages/promptable/src/tracing.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface TraceConfig {
 }
 
 const defaultConfig: TraceConfig = {
-  serverUrl: "http://localhost:3000/trace",
+  serverUrl: "http://localhost:3000/api/traces",
```
Comment (cfortuner) [inline packages/promptable/src/tracing.ts:12]: 🔥

File: apps/web/src/server/api/routers/traces.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+
+import { createTRPCRouter, publicProcedure } from "../trpc";
+
+export const traceRouter = createTRPCRouter({
+    getTraces: publicProcedure.query(async ({ ctx }) => {
+        const data = await ctx.prisma.trace.findMany();
+        const traces: any[] = [];
+        for (const d of data) {
+          traces.push(JSON.parse(d.trace));
```
Comment (cfortuner) [inline apps/web/src/server/api/routers/traces.ts:10]: Let's wrap this in a try { } catch {} just in case?

File: apps/web/src/pages/api/traces.ts [hunk 3]
```diff
@@ -2,23 +2,26 @@ import type { NextApiRequest, NextApiResponse } from "next";
...
-  //   return;
-  // }
-
-  const query = api.chain.getTraces.getQueryKey();
-  //@ts-ignore
-  const data = query.data ?? [];
-  const chains: any[] = [];
-  for (const d of data) {
-    chains.push(JSON.parse(d.trace));
+  if (req.method === "POST") {
```
Comment (cfortuner) [inline apps/web/src/pages/api/traces.ts:9]: hey! just wondering why this handler if we also use trpc? is this just so we don't need to worry about setting up openapi integration?

Comment (justinliang1020) [reply apps/web/src/pages/api/traces.ts:9]: i'll try rn to see if it's not too complex to set up openapi for it though

Comment (cfortuner) [reply apps/web/src/pages/api/traces.ts:9]: I've done this with openapi and can help here to save you time. Don't worry about it for now. This is good!

**Review by cfortuner — APPROVED** [review]
great work!

**Pull request merged**